data/formats-data.js

```diff
@@ -192,7 +192,7 @@ exports.BattleFormatsData = {
 		tier: "PU"
 	},
 	ekans: {
-		randomBattleMoves: ["coil","gunkshot","seedbomb","glare","suckerpunch","aquatail","earthquake","rest","rockslide"],
+		randomBattleMoves: ["coil","gunkshot","suckerpunch","aquatail","earthquake","rest"],
```

Comment:
Aqua Tail only hits Ground, but Ekans isn't staying in on Ground Pokemon, so it's useless. I think Glare is okay.

Switch Aqua Tail for Glare.